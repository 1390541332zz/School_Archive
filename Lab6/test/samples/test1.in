encrypt this is my phrase.
decrypt gsrh rh nb ksizhv.
