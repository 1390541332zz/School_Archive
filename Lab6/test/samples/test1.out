encrypted: gsrh rh nb ksizhv.
decrypted: this is my phrase.
